### PR TITLE
RFC4180: Add ContentType for text/csv

### PR DESCRIPTION
**Pull request by JLLeitschuh**
This Content Type is part of this RFC:
https://tools.ietf.org/html/rfc4180

### CODE REVIEW

File: ktor-http/src/io/ktor/http/ContentTypes.kt [hunk 1]
```diff
@@ -216,4 +217,4 @@ fun ContentType.withCharset(charset: Charset) = withParameter("charset", charset
 /**
  * Extracts a [Charset] value from the given `Content-Type`, `Content-Disposition` or similar header value.  
  */
-fun HeaderValueWithParameters.charset() = parameter("charset")?.let { Charset.forName(it) }
\ No newline at end of file
+fun HeaderValueWithParameters.charset() = parameter("charset")?.let { Charset.forName(it) }
```
Comment (JLLeitschuh) [inline ktor-http/src/io/ktor/http/ContentTypes.kt]: I created this PR with the github UI (didn't even check out the code locally). It seems to automatically add these when you do that. Sorry if this is an annoyance.

File: ktor-http/src/io/ktor/http/ContentTypes.kt [hunk 2]
```diff
@@ -184,6 +184,7 @@ class ContentType private constructor(val contentType: String, val contentSubtyp
         val Any = ContentType("text", "*")
         val Plain = ContentType("text", "plain")
         val CSS = ContentType("text", "css")
+        val Csv = ContentType("text", "csv")
```
Comment (cy6erGn0m) [inline ktor-http/src/io/ktor/http/ContentTypes.kt]: I'd say this need to be `CSV` as we already have abbreviations uppercased

Comment (JLLeitschuh) [reply ktor-http/src/io/ktor/http/ContentTypes.kt]: Fixed

**Comment by cy6erGn0m** [comment]
Thanks!

**Pull request merged**